pkg/operator/controllers/checker/ingressreplica_test.go

```diff
@@ -0,0 +1,127 @@
+package checker
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"testing"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	operatorfake "github.com/openshift/client-go/operator/clientset/versioned/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+	arofake "github.com/Azure/ARO-RP/pkg/operator/clientset/versioned/fake"
+)
+
+func TestIngressReplicaChecker(t *testing.T) {
+	ctx := context.Background()
+
+	var defaultReplica int32 = 2
+	var zeroReplica int32 = 0
```

Comment:
These vars probably aren't needed for the test, I think you could get away with just using the int values directly in the tests since you are only using each var once.